package.json

```diff
@@ -4,15 +4,15 @@
   "description": "Artsy React(Native) components.",
   "main": "index.ios.js",
   "scripts": {
-    "bundle": "react-native bundle --config rn-cli.config.js --platform=ios --dev=false --entry-file=index.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -r Pod/Assets/assets/node_modules",
+    "bundle": "react-native bundle --config \"$(pwd)\"/rn-cli.config.js --platform=ios --dev=false --entry-file=index.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -rf Pod/Assets/assets/node_modules",
     "clean-example": "cd Example && xcodebuild -workspace Emission.xcworkspace -scheme Emission -destination 'platform=iOS Simulator,name=iPhone 6' clean",
     "type-check": "tsc --noEmit --pretty",
     "lint": "tslint 'src/**/*.{ts,tsx}'",
     "lint-fix": "npm run lint -- --fix",
     "ci": "npm run type-check && npm run lint && npm run test -- --runInBand",
     "danger": "danger",
-    "start": "npm run clean-example && concurrently --kill-others 'npm run start-packager' 'npm run storybook'",
```

Comment:
Was there a reason that storybooks were removed from the app? I don't see a mention in the PR body